network-api/networkapi/buyersguide/models.py

```diff
@@ -76,11 +76,17 @@ class BuyersGuideProductCategory(models.Model):
         help_text='Description of the product category. Max. 300 characters.',
         blank=True
     )
+
     featured = models.BooleanField(
         default=False,
         help_text='Featured category will appear first on Buyer\'s Guide site nav'
     )
 
+    hidden = models.BooleanField(
+        default=False,
```

Comment:
due to the default being `False`, we don't need an explicit migration function that ensures all pre-existing categories get the correct value assigned.